scripts/init_letsencrypt.sh

```diff
@@ -7,12 +7,12 @@ set -o allexport
 source .env
 set +o allexport
 
-CONFIG_PATH="${CONFIG_PATH:-'./'}"
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
```

Comment:
```suggestion
SCRIPT_DIR=$(dirname "$(realpath "$0")")
```
would be a bit more elegant?

Reply:
I am not sure I am the correct person to ask. In any case, no objection from me.